src/index.js

```diff
@@ -1,53 +1,37 @@
 import './styles.css';
 import status from './status.js';
+import display from './display.js';
+import removeItem from './removeItem.js';
+import clearAll from './clearAll.js';
 
-let listItem = [
-  {
-    completed: false,
-    description: 'Learn how to use proper ES6 syntax',
-    index: 8,
-  },
-  {
-    completed: false,
-    description: 'Use ES6 modules to write modular JavaScript',
-    index: 12,
-  },
-  {
-    completed: false,
-    description: 'Use webpack to bundle JavaScript',
-    index: 6,
-  },
-];
+let listItem = [];
 
-listItem = listItem.sort((a, b) => a.index - b.index);
-
-if (!localStorage.getItem('list')) {
-  localStorage.setItem('list', JSON.stringify(listItem));
-}
-const populate = () => {
-  let storeList;
-  if (localStorage.getItem('list')) {
-    storeList = JSON.parse(localStorage.getItem('list'));
+class List {
+  constructor(description) {
+    this.completed = false;
+    this.description = description;
+    this.index = listItem.length + 1;
+    this.id = `id${this.index}`;
   }
-  const page = document.getElementById('list');
-  storeList.forEach((element) => {
-    const div = document.createElement('div');
-    const checkbox = document.createElement('input');
-    if (element.completed) {
-      (div.style.textDecoration = 'underline line-through');
-    } else {
-      (div.style.textDecoration = 'none');
+}
+
+const add = () => {
+  document.getElementById('addBtn').addEventListener('click', () => {
+    const newItem = new List(`${document.getElementById('input').value}`);
```

Comment:
- [ ] To-do items are still input fields. This issue also affects the accessibility of your page. Assistive technologies such as screen readers will consider them as input fields instead of items in your list. You can use `<p>` tag and make it easily editable with `contenteditable` attribute. Please check [this link](https://www.w3schools.com/tags/att_global_contenteditable.asp) for more information.